### DISCUSSION TITLE
[Question][Hw0-1]

### DISCUSSION THREAD

**Discussion by Rsln-M** [prosyslab-classroom/cs348-information-security#650] [Q&A]
Name: Ruslan Masimov

Hello. I set up the environment for working with OCaml according to the instructions. But I get 1 problem in VScode which doesn't hinder compiling and running the file but still shows up. The title is: "No config found for file hello.ml. Try calling `dune build`.

**Comment by m-spitfire** [comment]
duplicate, see #196.
